### PR TITLE
Allow overriding of the DNS Resolver IP used by the PawlTransportProvider

### PR DESCRIPTION
**Pull request by jmortlock**
By default the Ratchet Pawl connector will use a DNS server address of 8.8.8.8, we have the requirement to use a different server which is internal to our network.

This patch adds a second parameter to the PawlTransportProvider constructor which allows specifying a custom DNS server IP address.

### CODE REVIEW

File: src/Thruway/Transport/PawlTransportProvider.php [hunk 1]
```diff
@@ -22,13 +24,19 @@ class PawlTransportProvider extends AbstractClientTransportProvider
...
 
     /**
+     * @var string
+     */
+    private $DNS;
+
+    /**
      * Constructor
      *
      * @param string $URL
```
Comment (RafaelKa) [inline src/Thruway/Transport/PawlTransportProvider.php]: @param is missing

**Comment by jmortlock** [comment]
@RafaelKa Thanks, have updated with the doctype parameter.

**Comment by cookiecavekeeper** [comment]
This would be really nice to have👍

**Comment by jarkt** [comment]
I use an own fork where I not pass the nameserver IP but the Resolver instance. So it's possible to use a cached Resolver.
But I think it's important to merge this functionality.

**Comment by davidwdan** [comment]
Closed in lieu of injecting the connector into the transport https://github.com/voryx/Thruway/issues/269

**Pull request closed without merging**